<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-DQ5VT7O" failures="1" tests="8" name="TestCases.TS002_Validate_Login_Functionality" time="66.923" errors="0" timestamp="2025-05-31T19:58:13 IST" skipped="0">
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC001_Verify_LoginWithValidCredientials" time="7.619"/>
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC002_Verify_LoginWithInValidCredientials" time="7.444"/>
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC003_Verify_LoginWithInValidEmailAndValidPassword" time="7.739"/>
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC004_Verify_LoginWithValidEmailAndInValidPassword" time="7.051"/>
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC005_Verify_LoginWithoutCredientials" time="7.114"/>
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC006_Verify_ElementPlaceHolders" time="5.808"/>
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC007_Verify_ForgottonPasswordLink" time="6.523"/>
  <testcase classname="TestCases.TS002_Validate_Login_Functionality" name="TC008_Verify_BrowserBackActionAfterLogin" time="17.625">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at TestCases.TS002_Validate_Login_Functionality.TC008_Verify_BrowserBackActionAfterLogin(TS002_Validate_Login_Functionality.java:226)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- TC008_Verify_BrowserBackActionAfterLogin -->
</testsuite> <!-- TestCases.TS002_Validate_Login_Functionality -->
